test/examples/bootstrap.js

```diff
@@ -133,6 +147,11 @@ before(async () => {
             if (container) {
                 container.style.display = 'none';
             }
+            const divScaleWidget = document.querySelectorAll('.divScaleWidget');
```

Comment:
Does it break something ?

Reply:
Sometimes, this division interfere with mouse.down + mouse.move.

Reply:
Ok :+1: